### PR TITLE
home and review page fixes

### LINKED ISSUE #36: Fix issues with current / (Home page)
The index/home page was migrated as a database entry page instead of it's own template file.  The news items are "static" rather than pulling from the RSS feed.  The current dev site can be viewed at:

http://planetarymapping-dev.wr.usgs.gov/

The production version (in Java) can be viewed at

https://planetarymapping.wr.usgs.gov

The issues that need to be fixed:

- [x] Convert from static page to *.html template
- [x] Change @route for `/` to point at the new template
- [x] Include Javascript to load RSS feed in dynamically as per the example in the current production version
- [x] The bottom left of the main navigation (https://planetarymapping.wr.usgs.gov/Review) points at a missing page.  Use the "menu" template to create this bulleted list for navigation

### PR DESCRIPTION
**Pull request by amesamoyers**
Resolves #36 Removes RSS duplication when home is edited, adds /Review route and page, changes onlick events to href for homepage.  Includes html files for mongo db, dbInsert to update the mongo db and local-docker-compose to run the site locally.

### CODE REVIEW

File: pgm.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 import sys
 from functools import wraps
 from datetime import datetime
+from html.parser import HTMLParser
```
Comment (acpaquette) [inline pgm.py]: Unused import, could this get removed?

File: pgm.py [hunk 2]
```diff
@@ -99,12 +110,15 @@ def edit_page():
     form.page_title.data = file_title
     form.page_name.data = name
     form.page_content.data = file_markup
-    return render_template('edit.html', form = form, menu = menu)
+
+    print(form.page_content.data)
```
Comment (acpaquette) [inline pgm.py]: Old print that could be removed

File: pgm.py (hunk 1 above)
Comment (amesamoyers) [reply pgm.py]:
```suggestion
```

Comment (amesamoyers) [reply pgm.py]: @acpaquette thanks for spotting, will remove.

File: pgm.py (hunk 2 above)
Comment (amesamoyers) [reply pgm.py]:
```suggestion
```

**Comment by amesamoyers** [comment]
@acpaquette -- incorporated suggested changes

**Comment by acpaquette** [comment]
:+1: I can't merge this but it looks good to me. Go ahead and hit the button

**Pull request merged**